pages/content/posts/localizing-your-website.md

```diff
@@ -0,0 +1,33 @@
+
+Machine translations are not fully capable of capturing the nuances and subtleties of languages; they can translate but can't localize. Translations often result in the wrong word selection, translations of phrases that shouldn’t be translated (hot dog is "hot dog" everywhere; no one wants to eat a "_<span lang="fr-fr">chien chaud</span>_"), inconsistency in word selection throughout paragraphs, and even completely wrong translations.
+
+Just as in our previous example of the missing `alt` attributes, if you provide manual translations for a portion of your website but not everything, again a section of your content is not accessible.  Perhaps you’ve translated the content but not the interface, or vice versa, or perhaps you’ve only translated part of the content.  The user is seeing content on your page in two different languages but expects to only see one. When sections of your site are manually translated without providing internationalization features, such as the `lang` attribute on the HTML element or on translated content, machine translation software becomes out of the question. Automated translation services are likely to pick up already-translated words and attempt to “translate” them again, or be unable to determine what the original language is.  “What happened?” the user will think. “Is the website broken?  I can’t understand half of this…I'll look elsewhere.”
+
+And that’s the real kicker.  Even if the user understands the original language, they would still think that something is wrong, which would provide a negative user experience and give your competitors an opportunity to gain your lost consumers.  However, if a user is requesting an alternate language, chances are high that they can’t understand the original language in the first place!  By leaving your product partially translated (or realistically, untranslated at all), you are essentially telling the users that speak other languages: “we don’t really care about you.”
+
+Another benefit of localizing your website is that it can improve search engine optimization (SEO).  Search engines are more likely to rank websites that are localized for different languages higher in search results, as they are seen as providing a better user experience for users who speak those languages, the same way that websites with accessibility considerations are ranked higher.  By localizing a website, you can increase your visibility in search results and have a greater chance at gaining more organic traffic.
+
+Of course, it is also important to _maintain_ your localizations, rather than performing them once and forgetting about them.  As you update your website by adding or modifying content, it's important to ensure the localized versions are also kept up-to-date and always accurately reflect the latest content.  This helps prevent users from encountering outdated or inaccurate information, which leads to a negative user experience and potentially drives users away from the website.
```

Comment:
```suggestion
Of course, it is also important to _maintain_ your localizations, rather than performing them once and forgetting about them.  As you update your website by adding or modifying content, it's important to ensure the localized versions are also kept up-to-date and always accurately reflect the latest content.  This helps prevent users from encountering outdated or inaccurate information. Providing updated, relevant, internationalized content leads to a positive user experience and builds customer loyalty for your website, product, and company.
```

Reply:
made the last sentence positive. "Here's what you should do!"